### PR TITLE
Finalise TAD landing page

### PR DESCRIPTION
**Pull request by akenzua**
[TP11527 ](https://maps.tpondemand.com/restui/board.aspx#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNzlGODk3MENDQkM0N0FCM0ZFQzkyNEVFODEyREM0M0EifQ==&boardPopup=userstory/11527/silent)

This PR tidied up the styling issues on the landing page to conform to the wireframe

### CODE REVIEW

File: Utils/layouts/head.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const AppHead = ({ path, t }) => {
         type="image/x-icon"
         href="/static/favicon.ico"
       ></link>
+      <link rel="stylesheet" href="/static/accordion.css" />
```
Comment (davidtrussler) [inline Utils/layouts/head.js:30]: Just wondering: aren't these styles provided by the Library? I'm just a little surprised to see them loaded here is all.

File: public/static/locales/cy/landing.json [hunk 2]
```diff
@@ -37,31 +37,31 @@
...
         },
         {
           "question": "I haven’t been offered any insurance yet – I don’t know if I need a specialist.",
-          "answer": "<article>Then it’s worth trying a mainstream provider AND a couple of specialists to compare prices and cover offered. You can do this yourself – or you can ask an Insurance Broker to help you. The box at the bottom of this screen <strong>Need help shopping around?</strong> will tell you how.</article>"
+          "answer": "<article class='accordion'><p>Then it’s worth trying a mainstream provider AND a couple of specialists to compare prices and cover offered. You can do this yourself – or you can ask an Insurance Broker to help you. The box at the bottom of this screen <strong>Need help shopping around?</strong> will tell you how.</p></article>"
```
Comment (davidtrussler) [inline public/static/locales/cy/landing.json:44]: There are a number of pieces of copy that do not have translations.

**Comment by davidtrussler** [comment]
I've added a couple of small comments on this.

In addition to those though:
- are the points raised in the TP card addressed in this work or is that still to do?
- do we need to consider, in the FAQs section, a default set of styles for a non-JS scenario?

Also, though it's outside the scope of this PR, the language link in the footer doesn't work as it does in the header.

File: public/static/locales/cy/landing.json (hunk 2 above)
Comment (akenzua) [reply public/static/locales/cy/landing.json:44]: Yeah they havent been supplied yet but i can use google to add something there for now.

**Comment by akenzua** [comment]
> I've added a couple of small comments on this.
> 
> In addition to those though:
> 
> * are the points raised in the TP card addressed in this work or is that still to do?
> * do we need to consider, in the FAQs section, a default set of styles for a non-JS scenario?
> 
> Also, though it's outside the scope of this PR, the language link in the footer doesn't work as it does in the header.

They are, apart from the extra ones just added in the comment section which I am working one right now. For the accordion to work without JS, we will have to create another ticket. @andreferreiradlw and I already discussed that.

File: public/static/locales/cy/landing.json (hunk 2 above)
Comment (davidtrussler) [reply public/static/locales/cy/landing.json:44]: OK, no problem. Better to get the correct copy than add a temporary one though so for the purpose of this PR I'd say to leave that as it is.

**Comment by davidtrussler** [comment]
> They are, apart from the extra ones just added in the comment section which I am working one right now. For the accordion to work without JS, we will have to create another ticket. @andreferreiradlw and I already discussed that.

OK, so
- should we wait for the stuff in the comments to be done before approving this PR or will it be a separate card?
- on the accordion: that's fine for this PR.

File: Utils/layouts/head.js (hunk 1 above)
Comment (akenzua) [reply Utils/layouts/head.js:30]: > > They are, apart from the extra ones just added in the comment section which I am working one right now. For the accordion to work without JS, we will have to create another ticket. @andreferreiradlw and I already discussed that.
> 
> OK, so
> 
> * should we wait for the stuff in the comments to be done before approving this PR or will it be a separate card?
> * on the accordion: that's fine for this PR.

Yeah. Done.

**Comment by akenzua** [comment]
> > They are, apart from the extra ones just added in the comment section which I am working one right now. For the accordion to work without JS, we will have to create another ticket. @andreferreiradlw and I already discussed that.
> 
> OK, so
> 
> * should we wait for the stuff in the comments to be done before approving this PR or will it be a separate card?
> * on the accordion: that's fine for this PR.

Yeah. Done now.

**Comment by ladamsatmaps19** [comment]
Cool. Read the comments. Looks OK to me

**Pull request merged**